automation/release_notes/generate.py

```diff
@@ -105,12 +105,14 @@ def run(self):
     def _generate_release_notes_from_commits(
         self, commits_for_highlights, commits_for_pull_requests
     ):
-        highlight_notes = self._generate_highlight_notes_from_commits(
-            commits_for_highlights
-        )
+        (
+            highlight_notes,
+            failed_parsing_commits,
+        ) = self._generate_highlight_notes_from_commits(commits_for_highlights)
         # currently we just put everything under features / enhancements
         # TODO: enforce a commit message convention which will allow to parse whether it's a feature/enhancement or
         #  bug fix
+        failed_commits = '\n'.join(failed_parsing_commits)
```

Comment:
You can just do that inside the template, no need for extra variable - not that it has any affect on performance or something just hurts readability a bit, a matter of style obviously